graphql-dgs-client/src/main/kotlin/com/netflix/graphql/dgs/client/DefaultGraphQLClient.kt

```diff
@@ -39,7 +39,7 @@ import reactor.core.publisher.Mono
  *    });
  */
 @Deprecated("This has been replaced by [CustomGraphQLClient], [CustomReactiveGraphQLClient] and [WebClientGraphQLClient]")
-class DefaultGraphQLClient(private val url: String) : GraphQLClient, MonoGraphQLClient {
+class DefaultGraphQLClient(private val url: String, private val customDeserializer: Map<Class<*>, (Any) -> Any?>? = null) : GraphQLClient, MonoGraphQLClient {
```

Comment:
DefaultGraphQLClient is already deprecated. I don't think we should make any changes to it.